constraints.txt

```diff
@@ -144,7 +144,7 @@ Markdown==3.6
 markdown-it-py==3.0.0
 MarkupSafe==2.1.5
 marshmallow==3.21.3
-matplotlib==3.9.1
+matplotlib==3.9.0
```

Comment:
Just in case someone wonders what happened here: the 3.9.1 release of Matplotlib [has been withdrawn](https://pypi.org/project/matplotlib/3.9.1/).